examples/tutorials/Exploring_Quantum_Chemistry_with_GDB1k.ipynb

```diff
@@ -79,7 +79,7 @@
     "id": "AssRCAgB2UJi"
    },
    "source": [
-    "The ntext step we want to do is load our dataset. We're using a small dataset we've prepared that's pulled out of the larger GDB benchmarks. The dataset contains the atomization energies for 1K small molecules."
+    "The next step we want to do is load our dataset. We're using a small dataset we've prepared that's pulled out of the larger GDB benchmarks. The dataset contains the atomization energies for 1K small molecules. The structures in this dataset are represented as 2D matrices that capture the 3D electrostatic structure of the molecules. The format of the dataset is in SDF (Structure Data File) format."
```

Comment:
In general, we don't merge in doc fixes without at least a paragraph of additional information. This isn't bad but a bit too short for merge